> ```
> tensorflow/python/client/tf_session_wrapper.cc:1188:42: error: use of undeclared identifier 'TF_CXX_VERSION'
>   m.def("get_cxx_version", []() { return TF_CXX_VERSION; });
> ```

I cannot reproduce this locally/OSS. 
Do we need to run something specific when we change this file https://github.com/tensorflow/tensorflow/pull/57468/files#diff-b9e4766c0f0124421c09e4c012e29af59d4c218ee9163f0baa63bfd100a2720f ?

--- a/tensorflow/python/platform/sysconfig.py
+++ b/tensorflow/python/platform/sysconfig.py
@@ -19,7 +19,7 @@ import platform as _platform
 
 from tensorflow.python.client import pywrap_tf_session
 from tensorflow.python.framework.versions import CXX11_ABI_FLAG as _CXX11_ABI_FLAG
-from tensorflow.python.framework.versions import CXX_VERSION as __CXX_VERSION
+from tensorflow.python.framework.versions import CXX_VERSION as _CXX_VERSION
 from tensorflow.python.framework.versions import MONOLITHIC_BUILD as _MONOLITHIC_BUILD
 from tensorflow.python.framework.versions import VERSION as _VERSION
 from tensorflow.python.platform import build_info
@@ -64,13 +64,13 @@ def get_compile_flags():
   flags.append('-I%s' % get_include())
   flags.append('-D_GLIBCXX_USE_CXX11_ABI=%d' % _CXX11_ABI_FLAG)
   cxx_version_flag = None
-  if __CXX_VERSION == 201103:
+  if _CXX_VERSION == 201103:
     cxx_version_flag = "--std=c++11"
-  elif __CXX_VERSION == 201402:
+  elif _CXX_VERSION == 201402:
     cxx_version_flag = "--std=c++14"
-  elif __CXX_VERSION == 201703:
+  elif _CXX_VERSION == 201703:
     cxx_version_flag = "--std=c++17"
-  elif __CXX_VERSION == 202002:
+  elif _CXX_VERSION == 202002:
     cxx_version_flag = "--std=c++20"
   if (cxx_version_flag):
     flags.append(cxx_version_flag)

